internal/utils/web.go

```diff
@@ -80,3 +82,25 @@ func OpenBrowser(url string) error {
 		return fmt.Errorf("unsupported platform %q", runtime.GOOS)
 	}
 }
+
+func SendRequest(req *http.Request, resp interface{}) (header http.Header, statusCode int, err error) {
```

Comment:
I had `MakeRequest` originally and changed it. I changed it to `SendRequest` because I thought the name was odd and because of the docstring in `Do`. I'm sure there is a reason why they didn't call it `Send`, I'm just not sure what that is... I changed it to `DoRequest` though...

```
// Do sends an HTTP request and returns an HTTP response, following
...
func (c *Client) Do(req *Request) (*Response, error) {
```